README.md

```diff
@@ -527,21 +527,24 @@ Server 2008 due to API usage.
 - :create: creates a task
 - :delete: deletes a task
 - :run: runs a task
+- :end: ends a task
 - :change: changes the un/pw or command of a task
 - :enable: enable a task
 - :disable: disable a task
 
 #### Attribute Parameters
-- name: name attribute, The task name.
+- task_name: name attribute, The task name. ("Task Name" or "/Task Name")
 - command: The command the task will run.
 - cwd: The directory the task will be run from.
 - user: The user to run the task as. (requires password)
 - password: The user's password. (requires user)
 - run_level: Run with limited or highest privileges.
-- frequency: Frequency with which to run the task. (hourly, daily, ect.)
+- frequency: Frequency with which to run the task. (hourly, daily, etc.)
```

Comment:
Random nit: Can you also change it to (:hourly, :daily, etc.) to hint that there is an enumeration involved.